lib/Horde/Imap/Client/Data/Acl.php

```diff
@@ -166,4 +166,19 @@ public function unserialize($data)
         $this->_rights = json_decode($data);
     }
 
+    /**
+     * @return array
+     */
+    public function __serialize()
+    {
+        return array(
+            'rights' => $this->_rights
```

Comment:
strictly speaking this is not compatible with the old serialization methods but https://www.php.net/manual/en/language.oop5.magic.php#object.serialize has to return an array and `$this->_rights` is `false|array`.